fa/GLOSSARY.md

```diff
@@ -0,0 +1,3 @@
+# ویرایشگر کد
+
+ویرایشگر کد یک برنامه کاربردی است که به شما امکان می دهد کد خود را ذخیره کنید تا بتوانید بعدا به آن بازگردید. شما می توانید در [بخش ویرایشگر کد](./code_editor/README.md) یاد بگیرید که چطور یکی از آن‌ها را دانلود کنید
```

Comment:
```suggestion
ویرایشگر کد یک برنامه کاربردی است که به شما امکان می‌دهد کد خود را ذخیره کنید تا بتوانید بعداً به آن بازگردید. شما می‌توانید در [بخش ویرایشگر کد](./code_editor/README.md) یاد بگیرید که چطور یکی از آن‌ها را دانلود کنید
```

Reply:
@Rahimz that should be applied in Crowdin and then I will update pull request (as we want to manage all translations there eventually).